JYH/2Day/가운데 글자 가져오기.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    const math = s.length / 2;
+    let answer = '';
+    if(s.length % 2 === 0){
+        answer = s[math-1] + s[math];
+    }
+    else {
+        answer = s[Math.floor(math)];
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(s) {
    const math = s.length / 2;
    let answer = '';
    
    if (s.length % 2 === 0) {
        answer = s[math - 1] + s[math];
    } else {
        answer = s[Math.floor(math)];
    }
    return answer;
}
```
- math는 뭘 의미하는걸까요? s 문자열의 가운데 위치를 의미하는 좋은 변수명이 없을까요? 좋은 아이디어가 있으면 알려주세요 🤗
- 코드의 문맥을 이해하기 쉽게 변수 선언부와 if/for문과 같은 로직을 줄바꿈으로 분리해주는게 좋습니다 ㅎㅎ
- 누락된 공백과 if ~ else문에서 else문은 굳이 줄바꿈하지 않으셔도 됩니다

Reply:
혹시 math말고 추천해주실 좋은 변수명이 있을까요? 저는 단순하게 sootza 이런거 떠올라서 추천을 받고싶어요! 줄바꿈 하는 것도 다른 선배님이 위에 링크를 올려주셔서 그거 꼭 필독하겠습니다!

Reply:
@jungyoonho128 질문에 `힌트`를 드리긴 했지만 가이드를 드려보자면 ㅎㅎㅎㅎ 문자열의 가운데 인덱스라는 의미의 `middleIndex` 정도가 좋아보여요 🤗 해당 값이 의미하는 바와 네이밍의 싱크가 맞춰지는게 가장 좋습니다.

Reply:
많은 고민을 해보도록하겠습니다 감사합니다!